README.md

```diff
@@ -0,0 +1,56 @@
+# Azure SDK for Python
+
+[![Python](https://img.shields.io/pypi/pyversions/azure-core.svg?maxAge=2592000)](https://pypi.python.org/pypi/azure/) [![BuildStatus](https://dev.azure.com/azure-sdk/public/_apis/build/status/46?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46&branchName=master) [![Build](https://travis-ci.org/Azure/azure-sdk-for-python.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-python) [![Dependencies](https://img.shields.io/badge/dependencies-analyzed-blue.svg)](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-python/dependencies/dependencies.html)
```

Comment:
Kill Travis, I removed the yaml file anyway

Reply:
Done.